src/index.js

```diff
@@ -15,19 +15,38 @@ AsyncRedis.createClient = (...args) => new AsyncRedis(args);
 const commandsToSkipSet = new Set(['multi']);
 // this is the set of commands to promisify
 const commandSet = new Set(commands.filter(c => !commandsToSkipSet.has(c)));
+// this is the set of commands that return a Multi
+const queueCommandSet = new Set(['batch', 'multi']);
+// this is the set of Multi commands to promisify
+const multiCommandSet = new Set(['exec', 'exec_atomic']);
+
+const promisify = function (object, method) {
```

Comment:
It should be possible to use `utils.promisify` now too: https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_original

(It's since v8 of node)